pandas/core/generic.py

```diff
@@ -3137,6 +3137,78 @@ def clip_lower(self, threshold, axis=None):
         subset = self.ge(threshold, axis=axis) | isnull(self)
         return self.where(subset, threshold, axis=axis)
 
+    def round(self, decimals, out=None):
+        """
+        Round a DataFrame to a variable number of decimal places.
+
+        .. versionadded:: 0.17.0
+
+        Parameters
+        ----------
+        decimals: int, dict, Series
```

Comment:
Can you add a space between decimals and the colon (numpy docstring standard specification)